theme/tutorial-sidebar.less

```diff
@@ -434,6 +434,18 @@
     top: calc(5.5rem + @mainMenuHeight);
 }
 
+/*********************************
+        Accordion Hints (details)
+*********************************/
+
+.tabTutorial details {
+    padding: .5rem .5rem .5rem 2rem;
```

Comment:
2rem left padding is pretty significant in the details view -- something like 10% of the whole width on my machine, which is notable when you're putting an image or block in a hint: 
![image](https://user-images.githubusercontent.com/5615930/176067169-202b759f-6467-41fd-86e3-0fb7c4db0c03.png)

```suggestion
    padding: .5rem;
```

feels a bit better to me personally:

![image](https://user-images.githubusercontent.com/5615930/176067255-b0b30181-a4cf-44f4-90c3-dc6ddf0e39c4.png)